### PR TITLE
Add storage folder to store data in development environment

### PR DESCRIPTION
**Pull request by rsalevsky**
This PR is needed to prepare #144.

### CODE REVIEW

File: crowbar_framework/storage/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (aplanas) [inline crowbar_framework/storage/.gitignore]: this will include .gitignore again, isn't?

Comment (rsalevsky) [inline crowbar_framework/storage/.gitignore]: It's like ignore everything except this file.

**Comment by aplanas** [comment]
+1

File: crowbar_framework/storage/.gitignore (hunk 1 above)
Comment (jdsn) [inline crowbar_framework/storage/.gitignore]: :+1:

**Comment by jdsn** [comment]
:+1:

**Pull request merged**